src/index.d.ts

```diff
@@ -0,0 +1,21 @@
+type parseOptions = {
+  hiddenPaths?: string[];
+  matchesNode?: boolean;
+  maxPathLength?: number;
+  maxSourceLines?: number;
+  useColor?: boolean;
+  verbose?: boolean;
+}
+
+declare class UML {
+  constructor(elements: object[]);
+}
+
+declare module 'plantuml-parser' {
+  export function parse(data: string, options?: parseOptions): UML;
+  export function parseFile(pattern: string[], options?: parseOptions, cb?: (err: Error, ast: object) => Promise<UML>): UML | Promise<UML>;
```

Comment:
In pull request #29 I replaced all occurrences of `AST` with `parser result`, because from our discussion i figure out that the term AST does not fit here. If you could rename this here to `result` as well, please.